save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -4,27 +4,25 @@
 
 package org.cqfn.save.core.plugin
 
+import org.cqfn.save.core.config.TestConfigSections
+
 import kotlinx.serialization.Serializable
 
 /**
- * Core interface for plugin configuration
+ * Core interface for plugin configuration (like warnPlugin/fixPLuin/e.t.c)
```

Comment:
```suggestion
 * Core interface for plugin configuration (like warnPlugin/fixPlugin/e.t.c)
```

Reply:
fixed

Reply:
Seems, like it still `fixPluin` and `e.t.c`